### PR TITLE
Finish "Utility Port"

### PR DESCRIPTION
**Pull request by ghost**
abrbrbr
Ports the rest of the paired items with those in the Utility Port, including departmental jackets. Also tweaks a few of the uniforms in the Utility port (such as making chem-haz uniforms actually acid-proof)

## About The Pull Request
Completes the "Utility Port"
-Adds departmental coats to go with utility uniforms
-Adds matching suits and hats to costumes
-Adds a couple other assorted suits from the port (Bartender winter coat, alt-sprite Bomber Jacket, Flannels, QM Jacket)
-Tweaks a few of the utility outfits already added (Such as actually acid-proofing the chem-haz uniform)
-Modifies security utility outfit in loadout, now the ones in loadout are newsec. Oldsec still exists in the files should it need spawning though.

## Why It's Good For The Game
Outfit variety, woo!

## Changelog
:cl:
add: Added Departmental Jackets! Available in departmental wardrobes or as a Job-restricted item in the loadout!
add: Added Flannels! Fairly well insulating against temperature, but only on the upper body.
add: Added the finishing suits and hats to the Victorian costumes, as well as a Samurai costume to go with the kamishimo - all available in the Autodrobe (costume vendor)
fix: Security Utility items are now recolored for new blue-sec.
fix: Now-renamed "_Chem-Haz_" Uniforms are properly acid-proof and won't just melt off of you.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
And btw they all should work with digi legs, I tested those after the screenshot

File: modular_skyrat/modules/customization/modules/clothing/under/utility_port/suits_port.dm [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
+	mutant_variants = NONE
+	togglename = "zipper"
+
+//Job Jackets
+/obj/item/clothing/suit/toggle/jacket/engi
+	name = "engineering jacket"
+	desc = "A comfortable jacket in engineering yellow."
+	icon_state = "engi_dep_jacket"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 20, "fire" = 30, "acid" = 45)
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/customization/modules/clothing/under/utility_port/suits_port.dm]:
```suggestion
	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, RAD = 20, FIRE = 30, ACID = 45, WOUND = 0)
```

File: modular_skyrat/modules/customization/modules/clothing/under/utility_port/suits_port.dm [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+	allowed = list(/obj/item/flashlight, /obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman, /obj/item/t_scanner, /obj/item/construction/rcd, /obj/item/pipe_dispenser, /obj/item/toy, /obj/item/storage/fancy/cigarettes, /obj/item/lighter)
+
+/obj/item/clothing/suit/toggle/jacket/sci
+	name = "science jacket"
+	desc = "A comfortable jacket in science purple."
+	icon_state = "sci_dep_jacket"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 10, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/customization/modules/clothing/under/utility_port/suits_port.dm]:
```suggestion
	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 10, BIO = 0, RAD = 0, FIRE = 0, ACID = 0, WOUND = 0)
```

File: modular_skyrat/modules/customization/modules/clothing/under/utility_port/suits_port.dm [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+/obj/item/clothing/suit/toggle/jacket/med
+	name = "medbay jacket"
+	desc = "A comfortable jacket in medical blue."
+	icon_state = "med_dep_jacket"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 50, "rad" = 0, "fire" = 0, "acid" = 45)
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/customization/modules/clothing/under/utility_port/suits_port.dm]:
```suggestion
	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 50, RAD = 0, FIRE = 0, ACID = 45, WOUND = 0)
```

File: modular_skyrat/modules/customization/modules/clothing/under/utility_port/suits_port.dm [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+/obj/item/clothing/suit/toggle/jacket/supply/head
+	name = "quartermaster's jacket"
+	desc = "Even if people refuse to recognize you as a head, they can recognize you as a badass."
+	icon_state = "qmjacket"
+
+/obj/item/clothing/suit/toggle/jacket/sec
+	name = "security jacket"
+	desc = "A comfortable jacket in security blue. Probably against uniform regulations."
+	icon_state = "sec_dep_jacket"
+	armor = list("melee" = 25, "bullet" = 15, "laser" = 30, "energy" = 10, "bomb" = 25, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 45)
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/customization/modules/clothing/under/utility_port/suits_port.dm]:
```suggestion
	armor = list(MELEE = 25, BULLET = 15, LASER = 30, ENERGY = 10, BOMB = 25, BIO = 0, RAD = 0, FIRE = 0, ACID = 45, WOUND = 0)
```

**Review by Gandalf2k15 — CHANGES_REQUESTED** [review]

**Comment by OrionTheFox** [comment]
Whoops, I copied those values and totally forgot to change them to thee right formatting. I'll get it cube later today

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**